### PR TITLE
Minor fix

### PR DESCRIPTION
**Pull request by charles-cowart**
This fix was left out of the last PR.

### CODE REVIEW

File: qiita_db/processing_job.py [hunk 1]
```diff
@@ -160,9 +166,7 @@ def run(self):
 
                         # determine if anything has changed since last poll
                         if results['Job_Id'] in self.processes:
-                            tmp = cmp(self.processes[results['Job_Id']],
-                                      results)
-                            if 0 != tmp:
+                            if self.processes[results['Job_Id'] != results:
```
Comment (ElDeveloper) [inline qiita_db/processing_job.py]: This is missing one closing bracket.

File: qiita_db/processing_job.py [hunk 2]
```diff
@@ -86,12 +86,18 @@ def __init__(self):
...
             value = search('<%s>(.*?)</%s>' % (element, element), snippet)
             if value:
                 results[element] = value.group(1)
             else:
-                raise AssertionError("%s element is missing!" % element)
+                missing_elements.append(element)
+
+        if missing_elements:
+            raise AssertionError("The following elements were not found: %s"
+                                 % str(missing_elements))
```
Comment (ElDeveloper) [inline qiita_db/processing_job.py]: Can you format the list as: `', '.join(missing_elements)` and remove the `str` cast.

File: qiita_db/processing_job.py (hunk 1 above)
Comment (charles-cowart) [reply qiita_db/processing_job.py]: thanks!

File: qiita_db/processing_job.py (hunk 2 above)
Comment (charles-cowart) [reply qiita_db/processing_job.py]: You got it.

**Pull request merged**